src/auto/settings.rs

```diff
@@ -83,7 +83,7 @@ pub trait SettingsExt: 'static {
 
     fn bind<P: IsA<glib::Object>>(&self, key: &str, object: &P, property: &str, flags: SettingsBindFlags);
 
-    //fn bind_with_mapping<P: IsA<glib::Object>>(&self, key: &str, object: &P, property: &str, flags: SettingsBindFlags, get_mapping: /*Unimplemented*/Fn(/*Ignored*/glib::Value, &glib::Variant) -> bool, set_mapping: /*Unimplemented*/Fn(/*Ignored*/glib::Value, &glib::VariantType) -> glib::Variant, user_data: /*Unimplemented*/Option<Fundamental: Pointer>);
+    //fn bind_with_mapping<P: IsA<glib::Object>, Q: Fn(&glib::Value, &glib::Variant) -> bool + 'static, R: Fn(&glib::Value, &glib::VariantType) -> glib::Variant + 'static>(&self, key: &str, object: &P, property: &str, flags: SettingsBindFlags, get_mapping: Q, set_mapping: R);
```

Comment:
Why is this not generated? Seems complete now

Reply:
I don't know either. I'll need to check gir output.

Reply:
It doesn't detect the user data apparently: "function `g_settings_bind_with_mapping`'s callback `get_mapping` without associated user data"

Reply:
Can you create an issue for gir about this?

Reply:
I need to check if it comes from the gir file first.

Reply:
Seems like it comes from gir... Opening an issue.